olca-app/src/org/openlca/app/editors/graphical/model/Node.java

```diff
@@ -55,13 +56,8 @@ public class Node extends MinMaxComponent {
 	 * Helper variable when exploring graph in CollapseCommand
 	 */
 	public boolean isCollapsing;
-	/**
-	 * Helper variable when exploring graph in isChainingReferenceNode and in
-	 * isOnlyChainingReferenceNode.
-	 */
-	public boolean wasExplored;
 	private final String comparisonLabel = Labels.name(getRefFlow());
-	private final Map<Integer, Boolean> buttonStatus = new HashMap<>();
+	private final Map<Side, Boolean> buttonStatus = new HashMap<>();
```

Comment:
a nice example that this is much cleaner now, ideally, it would be an EnumMap now